OpenKeychain/src/main/res/xml/usb_device_filter.xml

```diff
@@ -24,4 +27,12 @@
     <!-- Yubikey 4 OTP + U2F + CCID-->
     <usb-device class="11" vendor-id="4176" product-id="1031"/>
 
+
+    <!-- Nitrokey Pro-->
+    <usb-device class="11" vendor-id="8352" product-id="16648"/>
+    <!-- Nitrokey Storage-->
+    <!--<usb-device class="11" vendor-id="8352" product-id="16649"/>-->
+    <!-- Nitrokey Start-->
+    <!--<usb-device class="11" vendor-id="8352" product-id="16913"/>-->
```

Comment:
can you add a comment here why these two are not activated right now for future reference?